File: SDKs/Aspose.Slides-Cloud-SDK-for-.NET/src/Com/Aspose/Slides/Model/PresentationsMergeRequest.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class PresentationsMergeRequest {
    public List<string> PresentationPaths { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class PresentationsMergeRequest {\n");
      sb.Append("  PresentationPaths: ").Append(PresentationPaths).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Slides-Cloud-SDK-for-Android/Aspose-Slides-Cloud-SDK-for-Android/src/main/java/com/aspose/slides/model/PresentationsMergeRequest.java
package com.aspose.slides.model;

import java.util.*;
public class PresentationsMergeRequest {
  private List<String> PresentationPaths = new ArrayList<String>();
  /**
	 * getPresentationPaths
	 * Gets List<String>
	 * @return PresentationPaths
	 */
  public List<String> getPresentationPaths() {
    return PresentationPaths;
  }

	/**
	 * setPresentationPaths
	 * Sets List<String>
	 * @param PresentationPaths List<String>
	 */
  public void setPresentationPaths(List<String> PresentationPaths) {
    this.PresentationPaths = PresentationPaths;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class PresentationsMergeRequest {\n");
    sb.append("  PresentationPaths: ").append(PresentationPaths).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

